test/Identity.FunctionalTests/UserStories.cs

```diff
@@ -135,5 +135,10 @@ public class UserStories
             var resetPassword = await ResetPassword.CreateAsync(link, client, new DefaultUIContext().WithExistingUser());
             return await resetPassword.SendNewPasswordAsync(email, newPassword);
         }
+
+        internal static async Task<ChangePassword> ChangePasswordAsync(ChangePassword changePassword)
+        {
+            return await changePassword.ChangePasswordAsync($"!Test.Password1$", $"!Test.Password2$");
```

Comment:
A few small comments: 
* Pass in the old password and the new password as parameters.